examples/existing_vpc/README.md

```diff
@@ -0,0 +1,3 @@
+# Basic Example
```

Comment:
There is already a "Basic Example". I would rename this to "Existing VPC and subnets example"